sklearn/tree/_export.py

```diff
@@ -247,17 +248,21 @@ def get_color(self, value):
             color = list(self.colors["rgb"][np.argmax(value)])
             sorted_values = sorted(value, reverse=True)
             if len(sorted_values) == 1:
-                alpha = 0
+                alpha = 0.0
             else:
                 alpha = (sorted_values[0] - sorted_values[1]) / (1 - sorted_values[1])
         else:
             # Regression tree or multi-output
```

Comment:
so here it's never multi-output? if it is, we shouldn't take the first value, should we?

Reply:
Apparently, the multi-output case is when `value` is not an Iterable.

Reply:
O_o then what is it? that seems very shaky, is there another way we could check for multioutput? this right now seems very confusing.

Reply:
In this case, I prefer to unpack from the call-site:

https://github.com/scikit-learn/scikit-learn/blob/a2b85716728da2596ee2275cbea939497af9146d/sklearn/tree/_export.py#L277-L281

and leave `get_color` alone.

Reply:
So I let `get_color` alone :)
I added a comment to differentiate the regression case from the classification from single class